server/controllers/manage/bookingsController.ts

```diff
@@ -20,15 +21,29 @@ export default class BookingsController {
   }
 
   new(): RequestHandler {
-    return (req: Request, res: Response) => {
+    return async (req: Request, res: Response) => {
       const { premisesId } = req.params
       const { errors, errorSummary, userInput } = fetchErrorsAndUserInput(req)
 
-      return res.render(`bookings/new`, {
+      const crnArr = req.flash('crn')
+      if (crnArr && crnArr[0]) {
```

Comment:
Would `if (crnArr) {` be enough here? An empty array is falsy right?

Reply:
It would make sense if an empty array was falsy... but as an array is an object and an empty object is truthy it is unfortunately truthy.

Reply:
Ah poop. Could we do `if (crnArr?.length)`

Reply:
yes